### ISSUE TITLE
Accessibility of the main navigation & language picker

### ISSUE THREAD

**Issue by enumiton** [mdn/yari#6027]
### The main navigation
Currently the main menu has three `menu`s, with `menuitem`s in the dropdown. This is wrong, as the `menu` role is mainly meant for replicating the feeling of a native menu as shown below. This means that people using screen readers can have trouble navigating or understanding this menu.

<img width="476" alt="A screenshot of a menu dropdown in the Windows operating system" src="https://user-images.githubusercontent.com/41439661/164444455-95d69442-50e9-4a25-b646-58a9122f2e14.png">

Never mind that a menu shouldn't be used as main navigation, it also hasn't been implemented properly as none of the [required keyboard shortcuts](https://www.w3.org/TR/wai-aria-practices-1.2/#keyboard-interaction-12) have been implemented.

### The language picker
The language picker also uses the `menu` and `menuitem` roles. While it could be argued that using the `menu` role here is valid, the implementation of it is again not correct. While the ul containing all the options has the `menu` role, none of its descendants have a `menuitem` role.

This menu also has no keyboard shortcuts implemented, nor does it manage any kind of focus.

### URL

All pages.

### Reproduction steps

N/A

### Expected behavior

When pressing a button which has `aria-haspopup="menu"`, I expect my focus to be shifted to the menu and to be able to use all required keyboard shortcuts for menus and menuitems.

### Actual behavior

1. Using the arrow keys within the menu does **not** select any items.
2. Pressing tab focuses the next item instead of focusing the next element after the menu.
3. Using the Home, End, PageUp or PageDown keys does **not** select any items.
4. Pressing Escape does nothing.

### Device

Desktop

### Browser

Other (specify below)

### Browser version

Stable

### Operating system

Other (specify below)

### Anything else?

I propose the main navigation is changed to normal links and ul's without the use of menu roles or menuitems.
One way is to make the ul visually hidden but still accessible through keyboard focus or by mouse hover, much like the "skip to content" buttons you can encounter by using tab.

As for the language picker, the keyboard shortcuts should be implemented and focus should be managed. Or the `menu` role is replaced by something else completely.

### Suggested blog posts
https://inclusive-components.design/menus-menu-buttons/
https://adrianroselli.com/2017/10/dont-use-aria-menu-roles-for-site-nav.html

### Validations

- [X] I have read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
- [X] I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
- [X] I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mdn/yari/discussions).

**Comment by enumiton** [comment]
The theme picker on desktop also has no keyboard shortcuts and doesn't manage focus.
It could also be argued the `menuitem` role should be applied to the button itself while the li receives the role `presentation` or `none` since the button is the item you actually interact with.

**Comment by caugner** [comment]
This issue has meanwhile been addressed in https://github.com/mdn/yari/pull/6929 (following discussion in https://github.com/mdn/yari/issues/6870).
